### DISCUSSION TITLE
Run validation set with LSTMFCNClassifier and tensorboard

### DISCUSSION THREAD

**Discussion by TannerCypret** [alan-turing-institute/sktime#7256] [General]
I am using tensorboard with LSTMFCNClassifier and getting training data like i should. I instantiate it like this: 

`tensorboard_callback = tf.keras.callbacks.tensorboard(logdir,histogram_freq=1)`

`lstm = LSTMFCNClassifier(n_epochs=50, batch_size=4,callbacks=tensorboard_callback )`

When i call fit:

`lstm.fit(x_train,ytrain)`

I get good data from tensorboard for the training. How do I run a validation set so that I can get validation data with my training data. From the docs I tried:

`lstm.fit(X_train, y_train).predict(X_test)`

This still only gives me training data. Is there function call to the LSTMFCNClassifier that can run validation data so that it gets logged in tensorboard?

**Comment by fkiraly** [comment]
Can you explain what you want to do?

Use `tensorboard` for logging the validation data, `X_test` against `y_test`?
